Microsoft.Azure.CosmosRepository.Samples/WebTier.Integration.Tests/LanguagesControllerTests.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Azure.CosmosRepository;
+using Microsoft.Extensions.DependencyInjection;
+using WebTier.Integration.Tests.Factories;
+using WebTier.Models;
+using Xunit;
+
+namespace WebTier.Integration.Tests
+{
+    public class LanguagesControllerTests : IClassFixture<WebTierApplicationFactory>
+    {
+        private readonly HttpClient _client;
+        private readonly IRepositoryFactory _repositoryFactory;
+
+        public LanguagesControllerTests(WebTierApplicationFactory factory)
+        {
+            _client = factory.CreateClient();
+            _repositoryFactory = factory.Services.GetRequiredService<IRepositoryFactory>();
+        }
+
+        [Fact]
+        public async Task Post_Always_Creates_A_Language()
+        {
+            //Arrange
+            LanguageDto language = new LanguageDto()
```

Comment:
```suggestion
            var language = new LanguageDto
```

Either `var` on the left, or `new()` on the right - but please, no double explicit typing 😬 .

Reply:
Ah okay, I do prefer `var` but the editor config does not allow it I can use `new()` on the right hand side thought, will tidy up.